frontend/src/main/java/com/google/tripmeout/frontend/storage/InMemoryPlaceVisitStorage.java

```diff
@@ -1,10 +1,16 @@
 package com.google.tripmeout.frontend.storage;
 
+import com.google.tripmeout.frontend.PlaceVisitModel;
+import com.google.tripmeout.frontend.error.PlaceVisitAlreadyExistsException;
+import com.google.tripmeout.frontend.error.PlaceVisitNotFoundException;
+import com.google.tripmeout.frontend.error.TripNotFoundException;
+
```

Comment:
There's something weird here. Not sure why the `com.google.tripmeout.frontend` packages are getting split around the `java.*` imports. Please make sure to run the formatter in any case.